### PR TITLE
KAA-363: Endpoints are not notified when topic is deleted.

### PR DESCRIPTION
**Pull request by serye**

### CODE REVIEW

File: server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/service/TopicServiceImpl.java [hunk 1]
```diff
@@ -105,19 +107,21 @@ public TopicDto findTopicById(String id) {
...
+    public List<UpdateNotificationDto> removeTopicById(String id) {
         validateId(id, "Can't remove topic. Invalid topic id " + id);
         TopicDto topic = findTopicById(id);
+        List<UpdateNotificationDto> updateNotificationDtos = new LinkedList<>();
         if (topic != null) {
             List<EndpointGroup> groups = endpointGroupDao.findEndpointGroupsByTopicIdAndAppId(topic.getApplicationId(), id);
             if (groups != null && !groups.isEmpty()) {
                 for (EndpointGroup eg : groups) {
-                    endpointService.removeTopicFromEndpointGroup(eg.getId().toString(), id);
+                    updateNotificationDtos.add(endpointService.removeTopicFromEndpointGroup(eg.getId().toString(), id));
```
Comment (shpigor) [inline server/common/dao/src/main/java/org/kaaproject/kaa/server/common/dao/service/TopicServiceImpl.java]: rename updateNotificationDtos to notificationList

**Comment by ashvayka** [comment]
Please update PR name to match our convention

**Comment by shpigor** [comment]
Reviewed

**Comment by ashvayka** [comment]
LGTM

**Pull request merged**